docs/source/cost_functions.rst

```diff
@@ -92,3 +92,30 @@ Jacobian matrix  :math:`J_f` is given by:
      \end{array}
    \right]
  \end{align}
+
+Landmark Cost Function
+======================
+
+Let :math:`\mathbf{p}_o` be the global pose of the SLAM tracking frame at which
+a landmark with the global pose :math:`\mathbf{p}_l` is observed.
+The landmark observation itself is the measured transformation
+:math:`\mathbf{T}^m_{ol}` that was observed at time :math:`t_o`.
+As the landmark can be observed asynchronously, :math:`\mathbf{p}_o` is modeled
+in between two regular trajectory nodes :math:`\mathbf{p}_i, \mathbf{p}_j`.
+
+Using the notation introduced above, the weighted landmark cost function is:
+
+.. math::
+  \mathbf f( \mathbf{p}_o, \mathbf{p}_l) = 
```

Comment:
Please, change `f` in the `Relative Transform Error 2D` section to `f_{\text{relative}}` and then smth like:

`f_{\text{landmark}}(\mathbf{p}_l, \mathbf{p}_{from}, \mathbf{p}_{to}) = f_{\text{relative}}(Interpolate(\mathbf{p}_{from}, \mathbf{p}_{to},), \mathbf{p}_l)`

so that we can see all variables that this function depends upon.